configure

```diff
@@ -2778,6 +2778,7 @@ ac_compiler_gnu=$ac_cv_c_compiler_gnu
 
 
 
+
```

Comment:
Spurious Change?

Reply:
Yes, I sprinkled some ``dnl``s over the autoconf file.